Makefile

```diff
@@ -140,8 +142,10 @@ contributors:
 	git log --format='%aN <%aE>' | sort -fu > CONTRIBUTORS
 
 js:
+ifeq (go1.8, $(findstring go1.8, $(GO_VERSION)))
```

Comment:
The latest version of GopherJS indeed requires Go 1.8. Here's a link that says it:

https://medium.com/gopherjs/gopherjs-1-8-1-is-released-b4473712ab95#.4espdfrxt

See second paragraph.